CRITICAL root:test_questions_api.py:38 ___response code- 404